hapi/hedera-protobufs/services/response_code.proto

```diff
@@ -1593,4 +1593,26 @@ enum ResponseCodeEnum {
      * airdrop and whether the sender can fulfill the offer.
      */
     INVALID_TOKEN_IN_PENDING_AIRDROP = 369;
+
+    /**
+    * The provided fee exempt key list size exceeded the limit.
+    */
+    MAX_ENTRIES_FOR_FEKL_EXCEEDED = 370;
```

Comment:
We should probably add `MAX_ENTRIES_FOR_EXEMPT_KEYS_EXCEEDED` or something, since this error is user facing ?